### PR TITLE
Feat/avatar has icon

### PR DESCRIPTION
**Pull request by Crash--**
Replace avatar icon="left" 

Is there a way to pass, I don't know, an array to `find()` in order to manage in only one method all our components with `icon` props instead of duplicating the code for each one?

### CODE REVIEW

**Comment by ptbrowne** [comment]
> Is there a way to pass, I don't know, an array to find() in order to manage in only one method all our components with icon props instead of duplicating the code for each one?

```
for (let component of [Avatar, Button, Icon]) {
  root.find(component).forEach(...)
}
```

?

File: packages/cozy-codemods/src/transforms/replace-icons-svgr.js [hunk 1]
```diff
@@ -70,5 +70,36 @@ export default function replaceSvgrIcons(file, api) {
...
+        imports.ensure(
+          root,
+          {
+            default: `Icon`
+          },
+          `cozy-ui/transpiled/react/Icon`
+        )
+        iconAttr.value = `{<Icon icon={${componentName}Icon} />}`
+      }
+    })
```
Comment (JF-Cozy) [inline packages/cozy-codemods/src/transforms/replace-icons-svgr.js:103]: Est-ce que ce ne serait pas intéressant de mettre une petite doc / commentaire pour expliquer pourquoi on fait ça ?

Comment (ptbrowne) [reply packages/cozy-codemods/src/transforms/replace-icons-svgr.js:103]: A doc for who ?

**Comment by Crash--** [comment]
> for (let component of ['Avatar', 'Button', 'Icon']) {
  root.find(j.JSXOpeningElement, { name: { name: component }}).forEach(...)
}

It was not clear enough, my question was more about find() itself not about any wrapper. I'll read jscodemod doc / code to see how it works under the wood

File: packages/cozy-codemods/src/transforms/replace-icons-svgr.js (hunk 1 above)
Comment (JF-Cozy) [reply packages/cozy-codemods/src/transforms/replace-icons-svgr.js:103]: J'avais pas vu qu'on était dans `cozy-codemods` :)

**Comment by Crash--** [comment]
done in #1170

**Pull request closed without merging**